cppcache/src/LocatorListRequest.hpp

```diff
@@ -34,13 +34,16 @@ class Serializable;
 class LocatorListRequest : public ServerLocationRequest {
  private:
   std::string m_servergroup;
+  bool m_requestInternalAddress;
 
  public:
-  explicit LocatorListRequest(const std::string& servergroup = "");
+  explicit LocatorListRequest(const std::string& servergroup = "",
```

Comment:
I suspect this signature has some really strange undefined behavior and now might be a time to fix it. We should probably go a step further and fix the default value assigned to a `std::string` reference. I think the keyword `explicit` can be dropped now since there is more than one argument. Maybe we default the first argument can convert but.. 🤷  What does clang-tidy say?